apps/frontend/assets/src/omg-page-api-detail-accounts/index.js

```diff
@@ -0,0 +1,356 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Link, withRouter } from 'react-router-dom'
+import queryString from 'query-string'
+import moment from 'moment'
+
+import SortableTable from '../omg-table'
+import Modal from '../omg-modal'
+import { Avatar, Button, Breadcrumb, Icon, Id, Select } from '../omg-uikit'
+import TopNavigation from '../omg-page-layout/TopNavigation'
+import AccessKeyMembershipsProvider from '../omg-access-key/accessKeyMembershipsProvider'
+import AssignAccountToKeyModal from '../omg-assign-account-key-modal'
+
+const BreadContainer = styled.div`
+  margin-top: 30px;
+  color: ${props => props.theme.colors.B100};
+  font-size: 14px;
+`
+const TitleContainer = styled.div`
+  span {
+    margin-left: 10px;
+  }
+`
+const NameContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  span {
+    margin-left: 10px;
+  }
+`
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 50px;
+  position: relative;
+  white-space: pre-line;
+  width: 500px;
+  h3 {
+    margin-bottom: 20px;
+  }
+  .close-icon {
+    position: absolute;
+    right: 15px;
+    top: 15px;
+    cursor: pointer;
+    color: ${props => props.theme.colors.S400};
+  }
+  .modal-buttons {
+    display: flex;
+    flex-direction: row;
+    margin-top: 20px;
+    button:first-child {
+      margin-right: 10px;
+    }
+  }
+`
+const RemoveCell = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+  width: 30px;
+  height: 30px;
+  margin-left: auto;
+  margin-right: auto;
+  transition: all 200ms ease-in-out;
+  :hover {
+    background-color: ${props => props.theme.colors.R300};
+    color: white;
+    cursor: pointer;
+  }
+`
+
+const KeyDetailAccountsPageView = withRouter(({
+  memberships,
+  membershipsLoading,
+  updateRole,
+  removeAccount,
+  refetch,
+  match: { params }
+}) => {
+  const { keyType, keyId } = params
+
+  if (memberships && !Array.isArray(memberships)) {
+    memberships = [memberships]
+  }
+
+  const [ roleModalOpen, setRoleModalOpen ] = useState(false)
+  const [ deleteModalOpen, setDeleteModalOpen ] = useState(false)
+  const [ assignAccountToKeyModal, setAssignAccountToKeyModal ] = useState(false)
+  const [ modalContent, setModalContent ] = useState({})
+  const [ loading, setLoading ] = useState(false)
+
+  const columns = [
+    { key: 'account.name', title: 'NAME', sort: true },
+    { key: 'account.id', title: 'ID', sort: false },
+    { key: 'role', title: 'ACCOUNT ROLE', sort: false },
+    // { key: 'account.parent_id', title: 'ASSIGNED BY', sort: false },
```

Comment:
wanna leave it here? or just forgot to remove.

Reply:
yea.. was gona do the 'assigned by' as seperate pr